lib/companies/schema/company.ex

```diff
@@ -13,7 +13,8 @@ defmodule Companies.Schema.Company do
     field :url, :string
 
     belongs_to :industry, Industry
-    has_many :jobs, Job
+    has_many :jobs, Job, defaults: [removed_pending_change_id: nil]
+    belongs_to :removed_pending_change, PendingChange
```

Comment:
Perhaps a comment on what the `removed_pending_change` does. It's not entirely clear when seeing the schema definition.

Reply:
Done